web/src/i18n/locales/en.json

```
@@ -9,6 +9,10 @@
     " 吗？": "?",
     " 秒": "s",
     " 秒。": " seconds.",
+    "，当前无生效订阅，将自动使用钱包": ", no active subscription. Wallet will be used automatically.",
```

Biome (2.4.10)

[error] The key ，当前无生效订阅，将自动使用钱包 was already declared.

(lint/suspicious/noDuplicateObjectKeys)